integration_tests/src/daphne.rs

```diff
@@ -10,11 +10,11 @@ use serde_json::json;
 use testcontainers::{clients::Cli, GenericImage, RunnableImage};
 use url::Url;
 
-const DAPHNE_HELPER_IMAGE_NAME_AND_TAG: &str = "cloudflare/daphne-worker-helper:sha-f6b3ef1";
+const DAPHNE_HELPER_IMAGE_NAME_AND_TAG: &str = "cloudflare/daphne-worker-helper:sha-4c612db";
```

Comment:
Reviewer note: This tag was created manually. It matches the commit that was checked out when the image was built. In the future we will add publishing the container to CI.